src/middleware/routeHandler.js

```diff
@@ -162,21 +179,46 @@ const attachRollbackRouteHandlerWrapper = (routeHandler) => async (
   await routeHandler(req, res, next).catch(async (err) => {
     try {
       if (shouldUseGitFileSystem) {
-        await backOff(() => {
-          const rollbackRes = gitFileSystemService
-            .rollback(siteName, originalCommitSha)
+        await backOff(async () => {
+          const rollbackRes = await gitFileSystemService
+            .rollback(siteName, originalStagingCommitSha, STAGING_BRANCH)
+            .andThen(() =>
+              gitFileSystemService.rollback(
+                siteName,
+                originalStagingLiteCommitSha,
+                STAGING_LITE_BRANCH
+              )
```

Comment:
Hmm should we separate this into 2 different rollback operations? Otherwise we might try to rollback staging again even though it's already succeeded, though I don't think this has any bad effects

Reply:
2e06bec48fd39e96336769aa4ca9b5b70ac81d19